src/App.js

```diff
@@ -1,25 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Same here, As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).), Please make the App a class-based component instead of a functional component.